website/src/theme/BlogPostItem/styles.module.css

```diff
@@ -5,9 +5,12 @@
  * LICENSE file in the root directory of this source tree.
  */
 
-.blogPostTitle {
+ .blogPostTitle {
   font-size: 1rem;
 }
+.blogPostPageTile{
```

Comment:
Still there is a minor issue when rendering long title in the individual blog pages. Please open a separate PR and address that.